tests/python/frontend/onnx/test_forward.py

```diff
@@ -5383,6 +5375,12 @@ def verify_eyelike(indata, dynamic=False):
     ]
 }
 
+flaky_target_skips = {
```

Comment:
Looking at this test: https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-13163/2/tests

It appears the issue is the `nearbyint` rounding, which is only supported for llvm targets.  

What is the expected rounding mode when using nearbyint. Perhaps you can just use the corresponding rounding function directly.

Reply:
See my comment above